modules/pref/src/main/Pref.scala

```diff
@@ -90,7 +90,13 @@ case class Pref(
         SoundSet.allByKey get value map { s =>
           copy(soundSet = s.key)
         }
-      case "zen"          => copy(zen = if value == "1" then 1 else 0).some
+      case "zen"          =>
+        copy(zen = 
+          value match 
+            case "1" => 1
+            case "2" => 2
+            case "3" => 3
```

Comment:
3? in which case do we want to set it to 3?

Reply:
Hi,

I was trying to do as you said here : https://github.com/lichess-org/lila/issues/12537#issuecomment-1592969351

To use 4 values of zen, while displaying 3 in the preference menu/ui. The user, if let's say he's in a puzzle and press 'z' while on value '2', would be having it's zen set to 3.